src/main/java/data/Vendor.java

```diff
@@ -97,6 +95,16 @@ public void addAccount(Account account) {
     return accountList;
   }
 
+  public Account getAccountById(String accountId) {
+    for (Account account : accountList) {
+      if (account.getAccountID().equals(accountId)) {
+        return account;
+      }
+    }
+    // returns null when account isn't found in the list.
```

Comment:
nit: This should be a JavaDoc comment above this method instead of an in-method comment here